### ISSUE TITLE
Getting `Blocked loading mixed active content` on Firefox when hosting app over HTTPS.

### ISSUE THREAD

**Issue by FF-AntiK** [billyb2/bootleg_networking#3]
I'm getting the `Blocked loading mixed active content` error when my WASM-Client tries to connect to the server.
Is there a way to use SSL in the server implementation?
If not, is there a way to trick Firefox into establishing the WebRTC connection anyways?

**Comment by billyb2** [comment]
You're serving the WASM files over HTTPS  too?

**Comment by FF-AntiK** [comment]
Yes, if I won't, Firefox would complain about gamepad support, which only works on secure connections.

**Comment by billyb2** [comment]
Honestly, I'm not too sure as to what to do. I'm not very knowledgable of browser networking, though it definetly sounds like an issue with the naia part of the codebase. Have you seen any similar issues there?

**Comment by FF-AntiK** [comment]
No Problem. I'll play with naia directly and report any findings here.

**Comment by FF-AntiK** [comment]
Hey, so I managed to setup a proxy to route secure traffic to the unsecured WebRTC server.
But that was only one part of the problem.
To make it work the `naia-client` needs to know the public WebRTC URL.
This is only possible in the latest `naia` release which uses `naia-socket 0.8.0`!

**Comment by billyb2** [comment]
I've been pretty busy with university lately, but I'd absolutely love to upgrade to the latest naia version! If someone would want to work on a PR for the upgrade, I'd be very happy to review and merge it. If not, I'll probably work on one at the end of my semester.
